src/Plugin/QuickForm/SaatenUnionSpraying.php

```diff
@@ -0,0 +1,427 @@
+    parent::__construct($configuration, $plugin_id, $plugin_definition, $messenger);
+    $this->entityTypeManager = $entity_type_manager;
+    $this->logger = $logger;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('messenger'),
+      $container->get('entity_type.manager'),
+      $container->get('logger'),
+    );
+  }
+
+/**
+ * Retrieves a sorted list of active, non-admin user labels.
+ *
+ * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
+ *   The entity type manager.
+ *
+ * @return array
+ *   An array of user labels indexed by user ID and sorted alphabetically.
+ */
+  protected function getUserOptions(EntityTypeManagerInterface $entityTypeManager): array {
+    // Query active, non-admin users.
+    $userQuery = $entityTypeManager->getStorage('user')->getQuery();
+    $userQuery->accessCheck(TRUE);
+    $userQuery->condition('status', 1);
+    $userQuery->condition('uid', '1', '>');
+  
+    // Load users.
+    $userIds = $userQuery->execute();
+    $users = $entityTypeManager->getStorage('user')->loadMultiple($userIds);
+  
+    // Build user options.
+    $options = [];
+    foreach ($users as $user) {
+      $options[$user->id()] = $user->label();
+    }
+    asort($options);
+  
+    return $options;
+  }
+  
+/**
+  * Helper function to get the labels of active equipment assets, sorted alphabetically.
+  *
+  * @return string[]
+  *   An array of equipment labels indexed by asset id and sorted alphabetically.
+  */
+protected function getActiveEquipmentLabels(): array {
+  try {
+    // Query active equipment assets.
+    $asset_storage = $this->entityTypeManager->getStorage('asset');
+    $asset_ids = $asset_storage->getQuery()
+      ->accessCheck(TRUE)
+      ->condition('status', 'active')
+      ->condition('type', 'equipment')
+      ->execute();
+
+    // Load equipment assets.
+    $equipments = $asset_storage->loadMultiple($asset_ids);
+
+    // Build equipment options.
+    $equipment_labels = array_map(function ($equipment) {
+      return $equipment->label();
+    }, $equipments);
+    natcasesort($equipment_labels);
+
+    return $equipment_labels;
+  } catch (\Exception $e) {
+    // Log the error and return an empty array.
+    $this->logger->error('Failed to load equipment assets: @message', [
+      '@message' => $e->getMessage(),
+    ]);
+    return [];
+  }
+}
+
+  /**
+   * Helper function to get the labels of active material assets, sorted alphabetically.
+   *
+   * @return string[]
+   *   An array of material labels indexed by asset id and sorted alphabetically.
+   */
+  protected function getMaterials(): array {
+    try {
+      // Get the entity storage for the 'asset' entity type.
+      $asset_storage = $this->entityTypeManager->getStorage('asset');
```

Comment:
Same - why is this try/catch block needed?

Reply:
It was accidentally added but it is not removed